src/containers/greeting/Greeting.css

```diff
@@ -48,6 +48,9 @@
 .greeting-image-div > img {
   max-width: 100%;
   height: auto;
+  padding-left: 400px;
```

Comment:
From what it looks like, I think your intention here was to space apart the text on the left from the image.

That intention is fine but doing it this way isn't the best way since it'll cause issues on different views that aren't your computer.

Here's the view on an iphone:
<img width="334" alt="Screenshot 2023-09-28 at 3 42 12 PM" src="https://github.com/sunkeiliu/personal-react-page/assets/11837954/423cb5d5-e9d2-49d8-994a-9dc787a3d383">

How I would suggest you do it is by 
-  flexbox rules on the parent container so that they space apart however much you want. So remove the `flex: 1 1` from the css rules for those components and see what happens. Also look up what it does. Consider using things like `justify-content: space-between`